tests/core/filter/manual/classcheckinlexicalorder.md

```diff
@@ -0,0 +1,22 @@
+@bender-tags: style, bug, 4.10.0
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea,sourcearea,toolbar,undo,clipboard,stylescombo,floatingspace
+
+Repeat below steps in all editors.
+
+----
+
+1. Select paragraph
+2. Open stylecombo
+3. Check what is displayed in styles combo
+
+**Expected:** Styles combo should display 3 different styles
+
+----
+
+1. Select entire paragraph
+2. Apply style `Marker 2` to the paragraph
+3. Do not change selection or reselect the same text
+3. Re-open stylescombo
```

Comment:
Typo: step '3.' is twice. You could just use '1.' in every step and let markdown parser to create correct lists.